### DISCUSSION TITLE
Hot to install to Surface 11 pro snapdragon?

### DISCUSSION THREAD

**Discussion by PHil050MA** [VirtualDrivers/Virtual-Display-Driver#190] [Q&A]
I tried several times. but whenever Add legacy devices, this driver is not certificated. so can't install driver.
how do i install this driver?

**Comment by bud3699** [comment]
If this is still relevent you need to install the arm64 driver. However on later versions of windows they changed the trusted publishers store, so for later versions of windows in arm64 need to turn off driver signature enforcement or test-signing
